browser/components/smartwindow/content/mentions.mjs

```diff
@@ -55,6 +55,15 @@ export class MentionDropdown {
 
     let itemIndex = 0;
 
+    // Show empty state
+    if (this.items.length === 0) {
+      const emptyState = document.createElement("div");
+      emptyState.className = "mention-section-header";
+      emptyState.textContent = "No tabs or pages found";
+      this.element.appendChild(emptyState);
+      return;
+    }
```

Comment:
Show an empty state for empty results.
<img width="69" height="72" alt="image" src="https://github.com/user-attachments/assets/99cf3a47-51e2-457a-8ef2-4c7d10c0a076" />